### ISSUE TITLE
Consider dropping support of Maven for Kotlin

### ISSUE THREAD

**Issue by sdelamo** [micronaut-projects/micronaut-platform#1075]
https://github.com/micronaut-projects/micronaut-platform/discussions/1074

**Comment by nedelva** [comment]
I am hoping you are not seriously considering dropping support for Maven!

**Comment by dstepanov** [comment]
The title should be `Consider dropping support of Maven for Kotlin`

**Comment by sdelamo** [comment]
> I am hoping you are not seriously considering dropping support for Maven!

We will continue to support Java + Maven. However, we are considering dropping Maven support for Kotlin.

✅ Java + Maven
❌ Java + Kotlin 

@nedelva would that be ok for you?

I  believe Kotlin users of Micronaut Framework should use Gradle.

**Comment by nedelva** [comment]
@sdelamo of course. I commented here solely because of the initial poorly worded title ("consider dropping support for Kotlin  AND Maven")

**Comment by roar-skinderviken** [comment]
I believe dropping Maven support for Kotlin is bad news for a lot of companies. My main customer has very little/no experience with Gradle, and all build templates are Maven-centric. Being responsible for choosing MN/Kotlin in a quite large architecture, I have mixed feelings regarding this suggestion.
